### ISSUE TITLE
Any way to change device id (or make the connection process not occur at boot)

### ISSUE THREAD

**Issue by joeynoodlesfang** [mongoose-os-libs/azure#1]
Hi, I've been trying to use this lib to change device id and send a message with the new id. Is there a way to do that currently without rebooting? 
Edit: Or, link 2+ devices simultaneously to the cloud using just one esp32?

**Comment by cpq** [comment]
Yeah there is a way to do it, but it is very tricky. You'd need to manually interfere with the mqtt flow.
Why rebooting is a problem anyway?

**Comment by joeynoodlesfang** [comment]
T_T any hints to where the flow needs to be interfered at? I've also tried manually initializing azure, ie: starting at false for enable and running a local version of the init_function. Didn't work. All the 3 ret's return true (cm dm shadow), but i'm guessing the mgos_mqtt_add_global_handler is the problem. I tried to create my own mgos_event_register_base, didn't correct the problem. So... where is the problem? My knowledge of MOS is quite superficial, so apologies if this seems obvious.
